test/functional/ex_cmds/map_spec.lua

```diff
@@ -50,40 +48,6 @@ n  asdf1         qwert
                  do the one thing]],
        helpers.exec_capture('filter the nmap'))
   end)
-
-  it('<Plug> mappings ignore nore', function()
```

Comment:
I'd suggest keep the test (as there is no timers or anything involved, time contrib is negligible)